prime-router/src/main/kotlin/transport/FTPSTransport.kt

```diff
@@ -190,17 +189,7 @@ class FTPSTransport : ITransport, Logging {
             try {
                 val fileInputStream: InputStream = ByteArrayInputStream(contents)
                 return ftpsClient.storeFile(fileName, fileInputStream)
```

Comment:
Revert these changes and use logger.error with the proper message and then the exception at the end.  Not sure what a proper message for the IO exception is, so use that old message for now.
logger.error("my message", e)